src/javascript/app/base/logged_in.js

```diff
@@ -30,25 +32,35 @@ const LoggedInHandler = (() => {
                 Client.doLogout({ logout: 1 });
             }
 
-            // redirect back
-            let set_default = true;
-            if (redirect_url) {
-                const do_not_redirect = ['reset_passwordws', 'lost_passwordws', 'change_passwordws', 'home', '404'];
-                const reg             = new RegExp(do_not_redirect.join('|'), 'i');
-                if (!reg.test(redirect_url) && urlFor('') !== redirect_url) {
-                    set_default = false;
+            // send marketing details to the backend, like when we open a new virtual account
+            // @see src/javascript/app/pages/user/new_account/virtual_acc_opening.js:82
+            const signup_device = LocalStore.get('signup_device') || /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent) ? 'mobile' : 'desktop';
+            const date_first_contact = LocalStore.get('date_first_contact') || toISOFormat(moment());
```

Comment:
if it's not available, let's not use today, we don't want to make assumptions and send wrong data. let's not send it if not available.

Reply:
You mean not send anything at all?

Reply:
yes, exclude it from the request